datajunction-server/tests/construction/fixtures.py

```diff
@@ -145,14 +145,14 @@ def build_expectation() -> Dict[str, Dict[Optional[int], Tuple[bool, str]]]:
         """basic.num_comments""": {
             None: (
                 True,
-                """SELECT  COUNT(1) AS cnt
+                """SELECT  COUNT(1) AS basic_DOT_num_comments
```

Comment:
I like how `_DOT_` is still surviving. When DJ is a trillion dollar project, it will be one of those quirks discussed at dinner tables worldwide.

Reply:
And there will be a fraction who will say that this is because of the project roots in the department of transportation.